### DISCUSSION TITLE
Changes feed of changed objects

### DISCUSSION THREAD

**Discussion by flohoff** [netbox-community/netbox#12280] [General]
Hi,
i am trying to push out changes in Netbox to the Switch infrastructure - I have the whole Ansible Inventory + Switchconfig stuff working (From Netbox) with VLANs, Port Security, EAP etc.

The question now is - How do i make some "notification" if something changed. From former inventory database i wrote i implemented a json http long polling interfaces which dumped all changed objects.

Does Netbox have anything like that e.g. some REST interface to poll the next changes after change XYZ?

Flo

**Comment by jeremystretch** [comment]
[Webhooks](https://docs.netbox.dev/en/stable/integrations/webhooks/) are generally the recommended solution for this sort of thing. NetBox will automatically send configured webhooks as objects are created, modified, and/or deleted.

Alternatively, you could poll the changelog API endpoint at `/api/extras/object-changes/`.
